app.js

```diff
@@ -0,0 +1,43 @@
+/* This creates a Hapi server which provides a RESTful API to allow Thimble
+ * and X-Ray Goggles to create/update/delete projects to/from a persistent
+ * store and also publish (make a project publicly available using the public
+ * access server) a project
+*/
+
+var Server = require('hapi').Server;
+var env = require('./environment.js');
+var authentication = require('./authentication');
+var routes = require('./routes');
+
+// Logging
+var hapiBunyan = require('hapi-bunyan');
+var log = require('./logger.js');
+
+var server = new Server();
+server.connection({ port: env.get('port') });
+
+// Configure authentication middleware
+authentication(server);
+
+// Configure server end-points
+routes(server);
```

Comment:
Similar thing here,  I recommend wrapping your routes in a registered plugin for the server